Network Policy - EndPort to Beta (#28860)
kubernetes/website: content/en/docs/concepts/services-networking/network-policies.md

The above rule allows any Pod with label `db` on the namespace `default` to communicate with any IP within the range `10.0.0.0/24` over TCP, provided that the target port is between the range 32000 and 32768.

The following restrictions apply when using this field:

> * As a beta feature, this is enabled by default. To disable the `endPort` field at a cluster level, you (or your cluster administrator) need to disable the `NetworkPolicyEndPort` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) for the API server with `--feature-gates=NetworkPolicyEndPort=false,…`.

Reviewer: Please consider wrap this long line since you are already onto it.
You can decide to wrap it a 80 or 120 characters or the end of a subsentence.
Long lines like this make future change tracking difficult and they ruin the days of the localization teams.

* The `endPort` field must be equal than or greater to the `port` field.
* `endPort` can only be defined if `port` is also defined.
* Both ports must be numeric.

{{< note >}}
Your cluster must be using a {{< glossary_tooltip text="CNI" term_id="cni" >}} plugin that
supports the `endPort` field in NetworkPolicy specifications.

> If your CNI does not support the `endPort` field and you specify a NetworkPolicy with that,

Reviewer 2: ```suggestion
If your [network plugin](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/)
does not support the `endPort` field and you specify a NetworkPolicy that specifies an `endPort`,
```
